core/primitives/src/version.rs

```diff
@@ -133,30 +126,23 @@ impl ProtocolFeature {
             ProtocolFeature::ForwardChunkParts => 45,
             ProtocolFeature::RectifyInflation => 45,
             ProtocolFeature::AccessKeyNonceRange => 45,
+            ProtocolFeature::AccountVersions => 46,
+            ProtocolFeature::TransactionSizeLimit => 46,
+            ProtocolFeature::AllowCreateAccountOnDelete => 46,
+            ProtocolFeature::FixStorageUsage => 46,
+            ProtocolFeature::CapMaxGasPrice => 46,
+            ProtocolFeature::CountRefundReceiptsInGasLimit => 46,
+            ProtocolFeature::MathExtension => 46,
```

Comment:
Are these features documented?

@khorolets @telezhnaya please, keep in mind the `AllowCreateAccountOnDelete` feature.

https://github.com/near/nearcore/blob/5a81fa01ba9402b02251e835699c75f9bb98b516/runtime/runtime/src/actions.rs#L519-L559

This seem to use a regular transfer action receipt, so Indexer for Explorer should be fine.